mne/report.py

```diff
@@ -765,7 +765,7 @@ def _add_figs_to_section(self, figs, captions, section='custom',
             # on some version mayavi.core won't be exposed unless ...
             from mayavi import mlab  # noqa, mlab imported
             import mayavi
-        except ImportError:
+        except:  # on some systems importing Mayavi raises SystemExit (!)
```

Comment:
...because apparently we can't be confident what subset of exceptions `mayavi` and its dependencies will throw, unfortunately.